README.md

```diff
@@ -65,22 +65,27 @@ This is a list of known adopters of Chaos Mesh. Some have already gone into prod
 - [DataStax Fallout](https://github.com/datastax/fallout)
 - [NetEase Fuxi Lab](https://fuxi.163.com/en/about.html)
 - [JuiceFS](https://juicefs.com/?hl=en)
-- [Meituan-Dianping](https://about.meituan.com/en)
-- [PingCAP](https://www.pingcap.com)
-- [Qihoo 360](https://www.360.cn/)
-- [Xpeng Motors](https://en.xiaopeng.com/)
+- [KingNet](https://www.kingnet.com/)
+- [Meituan Dianping](https://about.meituan.com/en)
+- [PingCAP](https://en.pingcap.com/)
+- [Apache Pulsar](https://pulsar.apache.org/)
+- [Qihoo 360](https://360.cn/)
+- [Qiniu Cloud](https://qiniu.com/en)
+- [Tencent](https://www.tencent.com/en-us)
+- [Vald](https://vald.vdaas.org/)
+- [WeBank](https://www.webank.com/)
+- [Xpeng](https://en.xiaopeng.com/)
```

Comment:
We should remove this list and add See Adopters, which links to the adopters.md file.

Reply:
👌🏽，I will update later.

Reply:
Done.